var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -54,10 +54,10 @@ class Conduit(Package):
     ###########################################################################
 
     variant("shared", default=True, description="Build Conduit as shared libs")
-    variant('test', default=True, description='Enable Conduit unit tests')
+    variant("test", default=True, description='Enable Conduit unit tests')
```

Comment:
for a follow-up: tests do not need to be a variant in Spack anymore.

They are controlled via `spack install --test=root conduit` which sets `self.run_tests`